pkg/cloudprovider/providers/aws/node.go

```diff
@@ -0,0 +1,33 @@
+package aws
+
+import (
+	"github.com/UKHomeOffice/keto/pkg/cloudprovider"
+)
+
+// Node returns an implementation of Node interface for AWS Cloud.
+func (c *Cloud) Node() (cloudprovider.Node, bool) {
+	return c, true
+}
+
+// GetKubeAPIURL returns a full URL to Kubernetes API.
+func (c *Cloud) GetKubeAPIURL() (string, error) {
+	instanceID, err := c.getInstanceID()
```

Comment:
This could be dried up a little. 2 getters in this change (and potentially other?) do the same thing internally.